Autocoders/Python/templates/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 ####
 # F prime CMakeLists.txt:
 #
-# SOURCE_FILES: combined list of source and autocoding diles
+# SOURCE_FILES: combined list of source and autocoding files
```

Comment:
I've thought this through a couple of times, and I'm pretty sure this is right...
c.f. https://hackmd.io/@jonathanmichel/By6Upb69P

Reply:
Yes this is the correct change.